api-model/src/main/java/io/enmasse/address/model/Status.java

```diff
@@ -63,19 +67,36 @@ public Status setMessages(Set<String> messages) {
         return this;
     }
 
+    public Status appendBrokerStatus(BrokerStatus brokerStatus) {
+        this.brokerStatuses.add(brokerStatus);
+        return this;
+    }
+
+    public Status clearBrokerStatuses() {
+        this.brokerStatuses.clear();
+        return this;
+    }
+
+    public Status setBrokerStatuses(List<BrokerStatus> brokerStatuses) {
```

Comment:
As setBrokerStatuses() retains a reference to the caller's list, which seems odd given  appendBrokerStatus() and clearBrokerStatuses() mutate it.    Maybe setBrokerStatus() should take a copy.  If you do this, change the ctor to take a copy too.